# HW 2

Just generic python setup first, including some helper functions to help with degree-radians conversion, and solving quadratic equations

In [1]:
import numpy as np
from numpy import sin, cos, tan, arctan2, sqrt

In [2]:
def rad(th):
    return th * np.pi/180

def deg(th):
    return th * 180/np.pi

def quadratic(a, b, c):
    return ((-b + sqrt(b**2 - 4*a*c)) / (2*a),
            (-b - sqrt(b**2 - 4*a*c)) / (2*a))

## Problem 1

The airplane is flying $v = 500$ mph at $30^\circ$ south of east. I'm going to use an angle of $\theta = -30^\circ$ so that I can use the usual sine / cosine formulas for angles that are counterclockwise from the $x$-axis, that is, north of east. I made the angle negative since it's clockwise / south of east.

In [3]:
v = 500.
th = rad(-30.)

In [4]:
vx = v * cos(th)
vy = v * sin(th)
print("vx = {:f}, vy = {:f}". format(vx, vy))

vx = 433.012702, vy = -250.000000


So in vector component form

$$
\vec v = 433 \,\text{mph}\, \hat i - 250 \,\text{mph}\, \hat j
$$

I could also have use the positive $30^\circ$ angle, and then just manually flipped the sign on the $v_y$ component, since the plan is heading south rather than north.